network-topology/nodes/sv-node/chronicle-release.md

```diff
@@ -11,21 +11,22 @@ The changes introduced in the Chronicle release are detailed in the sections bel
 To summarize the Chronicle release, the following points should be outlined:
 
 * **Restoration of Bitcoin's Original Protocol**: The Chronicle release aims to restore the original Bitcoin protocol by re-installing specific opcodes and removing listed restrictions, while also balancing stability for businesses that depend on the current state.
-* **Transaction Digest Algorithms**: The BSV Blockchain will now support both the Original Transaction Digest Algorithm (OTDA) and the BIP143 digest algorithm (with SIGHASH\_FORKID), ensuring compatibility and flexibility for developers and users. This restores the original Bitcoin transaction digest algorithm, enabling developers to have greater flexibilty in utilizing Bitcoin Script.
-* **Selective Malleability Restrictions:** The Chronicle Release removes restrictions that were put in place to prevent transaction malleability. To address concerns about the reintroduction of sources of transaction malleability, the application of malleability restrictions will depend on the interaction of the FORKID \[`0x40`] and RELAX \[`0x20`] sighash bits. Transactions signed without SIGHASH\_FORKID (or with RELAX enabled) will allow relaxed rules, removing strict enforcement of malleability-related constraints. This flexibility is agnostic to the number of signatures in a transaction. The restrictions relevant to the RELAX flag are:
+* **Transaction Digest Algorithms**: The BSV Blockchain will now support both the Original Transaction Digest Algorithm (OTDA) and the BIP143 digest algorithm, ensuring compatibility and flexibility for developers and users. This restores the original Bitcoin transaction digest algorithm, enabling developers to have greater flexibilty in utilizing Bitcoin Script. Usage of the OTDA will depend on the usage of the new CHRONICLE \[`0x20`] sighash bit.
```

Comment:
```suggestion
* **Transaction Digest Algorithms**: The BSV Blockchain will now support the Original Transaction Digest Algorithm (OTDA), in addition to the current BIP143 digest algorithm, ensuring compatibility and flexibility for developers and users. This restores the original Bitcoin transaction digest algorithm, enabling developers to have greater flexibility in utilizing Bitcoin Script. Usage of the OTDA will require setting the new CHRONICLE \[`0x20`] sighash flag.
```

Reply:
Makes it clear which of the two we're "adding" back. Clarifying that the flag must be set rather than unset.